### PR TITLE
Add Z-Wave.Me integration page

### PR DESCRIPTION
**Pull request by PoltoS**
## Proposed change

Added Z-Wave.Me integration page.
PR with the new integration: https://github.com/home-assistant/core/pull/61182

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65473
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3133
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Converting to draft. We'll add multiple parent PRs to a separate target branch before making the final PR to dev branch in core.

File: source/_integrations/zwave_me.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - light
+  - lock
+  - number
+  - sensor
+  - select
+  - siren
+  - switch
+---
+
+This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). This is the recommended Z-Wave integration for Home Assistant for owners of Z-Wave.Me controllers.
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]: I don't think we should write that this is the recommended Z-Wave integration for Z-Wave.Me controllers. Home Assistant doesn't recommend (nor discommend) this integration.

You're welcome to write the benefits of this integration that other Z-Wave integrations may not provide.

File: source/_integrations/zwave_me.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Quick start (Home Assistant including Supervisor)
+
+To add Z-Wave.Me to your installation, go to {% my integrations title="Configuration > Devices & Services" %} in the UI. Click the "Add integration" button in the bottom right and from the list of integrations, select "Z-Wave.Me" and follow the instructions shown.
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]: We can use the config flow template here which will provide this text and more for free.
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/zwave_me.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Z-Wave.Me Z-Way requires Z-Wave.Me RaZberry, RaZberry7, UZB1, Hub1 or any other Z-Wave.Me based controller.
+
+## Installing Z-Way
+
+Z-Wave.Me Z-Way runs on various platforms: Raspberry Pi OS, Linux, Windows. Installation instructions are available here: https://z-wave.me/z-way/download-z-way/
+
+## Migration to Z-Way
+
+To migrate from any Z-Wave controller to Z-Way, include Z-Way as a secondary controller (with security to let Z-Way learn the network key). Use Z-Way migration tool to make the controller primary in your network.
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
To migrate from any other Z-Wave controller to Z-Way, include Z-Way as a secondary controller (with security to let Z-Way learn the network key). Use the Z-Way migration tool to make the controller primary in your network.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/zwave_me.markdown [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - number
+  - sensor
+  - select
+  - siren
+  - switch
+---
+
+This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). This is the recommended Z-Wave integration for Home Assistant for owners of Z-Wave.Me controllers.
+
+## Quick start (Home Assistant including Supervisor)
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

**Comment by frenck** [comment]
Additional note, a PR against our brands repository is missing as well: <https://github.com/home-assistant/brands>

File: source/_integrations/zwave_me.markdown (hunk 1 above)
Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: Well, for owners of ZME controllers (Z-Way software) this is the only one. But okay, we can rephrase it

File: source/_integrations/zwave_me.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
+  - Sensor
+  - Siren
+  - Switch
+ha_release: '2021.12'
```
Comment (frenck) [inline source/_integrations/zwave_me.markdown]:
```suggestion
ha_release: 2022.2
```

**Comment by PoltoS** [comment]
> Additional note, a PR against our brands repository is missing as well: https://github.com/home-assistant/brands

Should it be in core_integrations folder?

File: source/_integrations/zwave_me.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  - light
+  - lock
+  - number
+  - sensor
+  - select
+  - siren
+  - switch
+---
+
+This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). It combines the performance and the power of the diagnostics tools built in Z-Way with the flexibility of Home Assistant. The integration brings all Z-Way devices in Home Assistant (Z-Wave, EnOcean, HTTP based and others).
```
Comment (frenck) [inline source/_integrations/zwave_me.markdown]:
```suggestion
This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). It combines the performance and the power of the diagnostics tools built-in Z-Way with the flexibility of Home Assistant. The integration brings all Z-Way devices in Home Assistant (Z-Wave, EnOcean, HTTP based, and others).
```

File: source/_integrations/zwave_me.markdown [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+{% include integrations/config_flow.md %}
+
+Note: You can use Z-Wave.Me UI with it's enhanced diagnostics tools together with the Home Assistant UI.
```
Comment (frenck) [inline source/_integrations/zwave_me.markdown]:
```suggestion
Note: You can use Z-Wave.Me UI with its enhanced diagnostics tools together with the Home Assistant UI.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
> Should it be in core_integrations folder?

Yes

**Comment by PoltoS** [comment]
> > Should it be in core_integrations folder?
> 
> Yes

Done https://github.com/home-assistant/brands/pull/3133

File: source/_integrations/zwave_me.markdown [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  - Siren
+  - Switch
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lawfulchaos, @Z-Wave-Me'
+ha_domain: zwave_me
+ha_platforms:
+  - binary_sensor
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown:21]:
```suggestion
  - binary_sensor
  - button
```

File: source/_integrations/zwave_me.markdown [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  - climate
+  - cover
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

File: source/_integrations/zwave_me.markdown [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+  - fan
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

File: source/_integrations/zwave_me.markdown [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+  - light
+  - lock
+  - number
+  - sensor
+  - select
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

File: source/_integrations/zwave_me.markdown [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  - siren
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

File: source/_integrations/zwave_me.markdown [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
+  - Sensor
+  - Siren
+  - Switch
+ha_release: 2022.2
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
ha_release: 2022.3
```

File: source/_integrations/zwave_me.markdown [hunk 14]
```diff
@@ -0,0 +1,54 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+featured: true
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]:
```suggestion
```

File: source/_integrations/zwave_me.markdown (hunk 9 above)
Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: Looks like we forgot this platform. We will add it.

File: source/_integrations/zwave_me.markdown (hunk 10 above)
Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: This one too

File: source/_integrations/zwave_me.markdown (hunk 12 above)
Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: And this one

File: source/_integrations/zwave_me.markdown (hunk 14 above)
Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: How to become features?

Comment (MartinHjelmare) [reply source/_integrations/zwave_me.markdown]: We evaluate the featured integrations once in a while and see if we should change it. It's a separate process from adding new integrations.

File: source/_integrations/zwave_me.markdown [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+ha_category:
+  - Binary Sensor
+  - Climate
+  - Cover
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown]: Please remove the categories we don't have platforms for, besides Hub.

Comment (PoltoS) [reply source/_integrations/zwave_me.markdown]: Done

File: source/_integrations/zwave_me.markdown [hunk 16]
```diff
@@ -0,0 +1,46 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+ha_category:
+  - Binary Sensor
+  - Button
+  - Climate
+  - Light
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_me.markdown:9]: I think we can keep Hub.
```suggestion
  - Hub
  - Light
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_integrations/zwave_me.markdown [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+
+Note: You can use Z-Wave.Me UI with its enhanced diagnostics tools together with the Home Assistant UI.
+
+## Hardware requirements
+
+Z-Wave.Me Z-Way requires Z-Wave.Me RaZberry, RaZberry7, UZB1, Hub1 or any other Z-Wave.Me based controller.
+
+## Installing Z-Way
+
+Z-Wave.Me Z-Way runs on various platforms: Raspberry Pi OS, Linux, Windows. Installation instructions are available here: https://z-wave.me/z-way/download-z-way/
```
Comment (frenck) [inline source/_integrations/zwave_me.markdown:43]:
```suggestion
Z-Wave.Me Z-Way runs on various platforms: Raspberry Pi OS, Linux, Windows. Installation instructions are available here: <https://z-wave.me/z-way/download-z-way/>
```

**Review by frenck — APPROVED** [review]
Thanks, @PoltoS 👍

**Pull request merged**